Add local mode to quickstart

nrfox opened the pull request:
Adds a section to the quickstart on running Kiali locally.

Fixes https://github.com/kiali/kiali/issues/8634

nrfox committed:
Add local mode to quickstart

jshaughn reviewed (approved):
@nrfox This looks good to me, did you have more to add, or just not assign a reviewer?

nrfox commented:
> This looks good to me, did you have more to add, or just not assign a reviewer?

I just forgot to assign a reviewer when I moved it out of draft but I will make one small update.

nrfox committed:
Remove example of downloading the binary

The exact URL may be different from the format in the docs and instructions may vary depending on the platform and the task is simple enough that users should be able to handle this.
